lib/commands/on-complete.js

```diff
@@ -1,9 +1,10 @@
 var assert = require('assert');
 var log = require('db-migrate-shared').log;
 
-module.exports = function (migrator, internals, callback, originalErr) {
+module.exports = function (migrator, internals, callback, originalErr, results) {
   if (typeof callback !== 'function') {
-    originalErr = originalErr || callback;
+    originalErr = callback;
```

Comment:
this change causing test failures. 

As I understand it, the difference between the two only happens when:
- `callback` is provided but is not a function
- `originalErr` is also provided

When would that ever occur? Attempting to debug via the tests but not easy going due to the child processes

Reply:
Remove this change please, this is interferring with core functionalities.